sound/soc/sof/ipc4-topology.c

```diff
@@ -2947,15 +2946,17 @@ static int sof_ipc4_route_setup(struct snd_sof_dev *sdev, struct snd_sof_route *
 	sroute->src_queue_id = sof_ipc4_get_queue_id(src_widget, sink_widget,
 						     SOF_PIN_TYPE_OUTPUT);
 	if (sroute->src_queue_id < 0) {
-		dev_err(sdev->dev, "failed to get queue ID for source widget: %s\n",
+		dev_err(sdev->dev,
+			"failed to get output queue ID for source widget: %s\n",
 			src_widget->widget->name);
 		return sroute->src_queue_id;
 	}
 
 	sroute->dst_queue_id = sof_ipc4_get_queue_id(src_widget, sink_widget,
 						     SOF_PIN_TYPE_INPUT);
 	if (sroute->dst_queue_id < 0) {
-		dev_err(sdev->dev, "failed to get queue ID for sink widget: %s\n",
+		dev_err(sdev->dev,
+			"failed to get input queue ID for sink widget: %s\n",
```

Comment:
I would use the same terminology as the code

"failed to get dst_queue_id ID for sink widget:"

Reply:
here the dst_queue_id is the source queue of the sink widget.
I'm not sure which is better, but if we refer to the widget then it is from the widget's pow?

Reply:
My point is that we don't need to add a new 'input' term that's not used anywhere else.
we only use 'input' and 'output' for formats.

Reply:
"failed to get src_queue_id ID from source widget: %s"
and
"failed to dst src_queue_id ID from sink widget: %s"

?